example/storybook/src/components/Forms/FormControl/index.stories.mdx

```diff
@@ -275,6 +301,594 @@ The following are the lists of all the libraries and packages the component reli
+                            Name
+                          </FormControl.Label.Text>
+                          <Input>
+                            <Input.Input
+                              type="text"
+                              py='$2'
+                              value='Wade Warren'
+                            />
+                          </Input>
+                        </VStack>
+                        <VStack space='xs'>
+                          <FormControl.Label.Text color='$text500'>
+                            Email
+                          </FormControl.Label.Text>
+                          <Input>
+                            <Input.Input
+                              type="text"
+                              py='$2'
+                              value='wadewarren@sample.com'
+                            />
+                          </Input>
+                        </VStack>
+                        <FormControl.Label.Text color='$text900' fontWeight='$normal'>
+                          Volume
+                        </FormControl.Label.Text>
+                        <Slider defaultValue={40}>
+                          <Slider.Track>
+                            <Slider.FilledTrack bg='$emerald700'/>
+                          </Slider.Track>
+                          <Slider.Thumb bg='$emerald700'/>
+                        </Slider>
+                        <HStack space='sm'>
+                          <Switch size='sm'/>
+                          <FormControl.Label.Text color='$text900' fontWeight='$normal' lineHeight='$md'>
+                            Dark Mode
+                          </FormControl.Label.Text>
+                        </HStack>
+                      </VStack>
+                      <Button
+                        bg='$emerald700'
+                        mt='$4'
+                        onPress={()=>{
+                          setShowModal(false);
+                        }}
+                      >
+                        <Button.Text>
+                          Save Changes
+                        </Button.Text>
+                      </Button>
+                    </FormControl>
+                  </Modal.Body>
+                </Modal.Content>
+              </Modal>
+            </Center>
+          </Wrapper>
+        );
+      }
+      `,
+      scope: {
+        Wrapper,
+        useState,
+        Box,
+        Center,
+        FormControl,
+        Input,
+        Button,
+        Slider,
+        Switch,
+        Modal,
+        Text: FormControlText,
+        Heading,
+        VStack,
+        HStack,
+      },
+      argsType: {},
+    }}
+  />
+</AppProvider>
+
+#### Form Control with Form Actions
+
+<AppProvider>
+  <CodePreview
+    showComponentRenderer={true}
+    showArgsController={false}
+    metaData={{
+      code: `
+        <FormControl flexDirection='row'>
+          <Button bg='$red700' mr='$4' py='$2.5' px='$3'>
```

Comment:
Check the hover and active state color of this button. It should be of red shade.

Reply:
checked, the color is the same for hover state.

Reply:
change it to darker shade